src/requireDatabaseInstance.ts

```diff
@@ -0,0 +1,34 @@
+import * as clc from "cli-color";
+import { getDefaultDatabaseInstance } from "./getDefaultDatabaseInstance";
+import * as utils from "./utils";
+
+/**
+ * Ensures that the supplied options have an instance set. If not, tries to fetch the default instance.
+ * @param options command options
+ */
+export async function requireDatabaseInstance(options: any) {
+  if (options.instance) {
+    return Promise.resolve();
+  }
+  try {
+    const instance = await getDefaultDatabaseInstance(options);
+    if (instance === "") {
+      return utils.reject(
+        `It looks like you haven't created a Realtime Database instance in this project before. Go to ${clc.bold.underline(
+          `https://console.firebase.google.com/project/${options.project}/database`
+        )} to create your default Realtime Database instance.`,
+        { exit: 1 }
+      );
+    }
+    options.instance = instance;
+    return Promise.resolve();
+  } catch (err) {
+    return utils.reject(
+      `Failed to get details for project: ${options.project}. Please look at firebase-debug.log for more details`,
```

Comment:
I'm not a fan of putting "look at firebase-debug.log" all over our codebase. Can it be removed please?

```suggestion
      `Failed to get details for project: ${options.project}.`,
```